packages/ecs-browser/src/ValueEditor.tsx

```diff
@@ -51,6 +51,11 @@ export const ValueEditor = ({
   const persistValue = useCallback(
     (e: Event | React.SyntheticEvent, value: string | null) => {
       e.preventDefault();
+
+      if (!setContractComponentValue) {
+        return;
+      }
+
```

Comment:
It should still be possible to edit local component values without `setContractComponentValue` function

Reply:
Won't this purely be client side then? I guess thats fine as well tho!

Reply:
Yes, it's for quickly editing client-only components in dev

Reply:
I agree we should remove this to support client-only editing.